connaisseur/validators/notrayv1/notary.py

```diff
@@ -47,8 +56,24 @@ def __init__(self, name: str, host: str, pub_root_keys: list, **kwargs):
         self.name = name
         self.host = host
         self.pub_root_keys = pub_root_keys
-        self.is_acr = kwargs.get("is_acr", False)
-        self.is_cosign = kwargs.get("is_cosign", False)
+        self.is_acr = is_acr
+        self.auth = auth
+        self.cert = self.__create_cert(cert) if cert else None
+
+    def __create_cert(self, cert: str):
```

Comment:
This sound like a certificate is generated, but it's actually only written. (I think this is where my initial confusion came from.) Renaming it to `__write_cert` would remove the ambiguity, but it's not too important.